tests/test_ZoneTransit.py

```diff
@@ -0,0 +1,36 @@
+from unittest import TestCase
+
+import libZoneTransit
+from pdb import set_trace as dbg
+
+class TestZoneTransit(TestCase):
```

Comment:
@maym2104 Ok for now but this should be segmented into many files and many functions ... I will show you.